File: backend/mailbook/praktika/models/sale.py
from django.db import models


class Sale(models.Model):
    discount = models.FloatField(null=True,verbose_name='Скидка')
    price = models.FloatField(null=True,verbose_name='Цена')
    amount = models.IntegerField(null=True,verbose_name='Количество')


    def __str__(self):
        return f'{self.price} р.'

File: backend/src/models/enteties/sale.ts
import {
    BaseEntity,
    Column,
    Entity,
    PrimaryGeneratedColumn
} from "typeorm"



@Entity('sale')
export class Sale extends BaseEntity {
    @PrimaryGeneratedColumn("uuid")
    id: string;

    @Column()
    discount: number

    @Column({
        type:"numeric"
    })
    price: number

    @Column()
    amount:number
}